System.Activities.Presentation/InternalApis/Clr/inc/AppContextDefaultValues.cs

```diff
@@ -0,0 +1,180 @@
+﻿// ==++==
+// 
```

Comment:
I am not finding this file in either the 4.7.2 or 4.8 TF branches. Where did this come from?